Header Header

uint32  no_obj  #number of object
RadarTarget[]   objs    #targets